### DISCUSSION TITLE
Load-balancer modeling plugin

### DISCUSSION THREAD

**Discussion by erjac77** [nautobot/nautobot#995] [Plugin (App) Ideas & Development]
Following the first Nautobot Community Meeting two weeks ago, I'm looking to start a discussion on modeling LB/ADC objects, possibly via a plugin. The initial idea would be to model components such as node/host, member/server, pool/cluster/farm and virtual server. We could also seek to associate subnets with VS (Virtual IPs) and Pools (SNAT Pools).

https://www.f5.com/services/resources/white-papers/load-balancing-101-nuts-and-bolts

**Comment by bpetrbok** [comment]
This seems to really be the only mention, even, of getting load-balancer data into Nautobot. Perhaps I haven't scoured the INET hard enough yet to find anyone else bringing this up  ...

I'd like, nay - LOVE - to get some traction on this plugin. This seems to be a large gap in network SoT.

**Comment by bpetrbok00** [comment]
Network to Code has release the load-balancer model with 3.0, we're already utilizing it - YAY!!
